src/main/java/lesson12Names/NameChanger.java

```diff
@@ -0,0 +1,28 @@
+package lesson12Names;
+
+import java.util.*;
+
+public class NameChanger {
+
+    public static void main(String[] args) {
+
+        ArrayList<String> namesList = new ArrayList<>(Arrays.asList
```

Comment:
Zmienna namesList:
1. Powinna być typu List, nie ArrayList
2. mogłaby się nazywać names. To, że jest to lista widzimy po typie, więc "List" w nazwie jest zbędne.